File: twosum.c
#include<stdio.h>
void findTwoSum(int arr[], int size, int target) {
    for (int i = 0; i < size - 1; i++) {
        for (int j = i + 1; j < size; j++) {
            if (arr[i] + arr[j] == target) {
                printf("Indices: [%d, %d]\n", i, j);
                return;
            }
        }
    }
    printf("No two elements add up to the target sum.\n");
}

int main() {
    int size, target;

    printf("Enter the size of the array: ");
    scanf("%d", &size);

    int arr[size];
    printf("Enter %d elements:\n", size);
    for (int i = 0; i < size; i++) {
        scanf("%d", &arr[i]);
    }

    printf("Enter the target sum: ");
    scanf("%d", &target);

    findTwoSum(arr, size, target);

    return 0;
}

File: twosum.py
def twoSum(nums, target):
    n = len(nums)
    for i in range(n - 1):
        for j in range(i + 1, n):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []


nums = [2, 7, 11, 15]
target = 9
result = twoSum(nums, target)
print("Indices of the two numbers:", result)
